canonical-web-and-design/maas-ui
# Add RAM column to new machine listing.

## Done
- Added RAM column to new machine listing. Also includes ScriptStatus icon + tooltip.

## QA
- Go to the new machine list and check that the RAM column looks like the old one.
- If the machine has failed memory tests, check that an error icon with a tooltip shows.

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1762
